packages/core/components/StatusMessage/StatusMessage.module.css

```diff
@@ -4,7 +4,7 @@
     width: 40vw; 
     height: 100%;
     left: 30px;
-    z-index: 1000;
+    z-index: 99999999999;
```

Comment:
Made the statusmessage component render at the same level as the modal so errors produced by a modal are visible